core/insertion_marker_manager.js

```diff
@@ -160,6 +160,14 @@ Blockly.InsertionMarkerManager.prototype.dispose = function() {
   }
 };
 
+/**
+ * Update the available connections for the top block. These connections can
+ * change if a block is unplugged and the stack is healed.
+ */
+Blockly.InsertionMarkerManager.prototype.updateAvailbleConnections = function() {
```

Comment:
typo: updateAvailbleConnections > updateAvailableConnections

Reply:
oh jeez, thanks for catching!